pandas/tests/reshape/merge/test_merge_asof.py

```diff
@@ -1006,6 +1006,18 @@ def test_merge_datatype_error(self):
         with pytest.raises(MergeError, match=msg):
             merge_asof(left, right, on='a')
 
+    def test_merge_datatype_categorical_error(self):
```

Comment:
Can you append `_raises` to the test name here?

Reply:
Sure. This was copied from `test_merge_datatype_error` (just above), though. Do I do that for that one too?

Reply:
Yea that should be fine to include here